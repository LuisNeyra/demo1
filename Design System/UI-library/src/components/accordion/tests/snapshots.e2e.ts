import { snapIt } from '../../../test';

const component = 'accordion';
const snap = snapIt(component);

snap('emtpy', `<blaze-accordion></blaze-accordion>`);
snap(
  'two headers and two panes',
  `<blaze-accordion>
      <blaze-accordion-pane header="Click me">
        This is an accordion
      </blaze-accordion-pane>
    </blaze-accordion>`
);
snap(
  'two headers and two panes',
  `<blaze-accordion>
      <blaze-accordion-pane open header="Click me">
        This is an accordion
      </blaze-accordion-pane>
    </blaze-accordion>`
);
