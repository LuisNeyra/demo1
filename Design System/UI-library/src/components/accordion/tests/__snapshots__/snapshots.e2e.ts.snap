// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`emtpy 1`] = `"<blaze-accordion class=\\"hydrated\\"><!----><div class=\\"c-card c-card--accordion\\"></div></blaze-accordion>"`;

exports[`two headers and two panes 1`] = `"<blaze-accordion class=\\"hydrated\\"><!----><div class=\\"c-card c-card--accordion\\"> <blaze-accordion-pane header=\\"Click me\\" class=\\"hydrated\\"><!----><button role=\\"heading\\" aria-expanded=\\"false\\" class=\\"c-card__control\\">Click me</button><section aria-hidden=\\"\\" class=\\"c-card__item c-card__item--pane\\"> This is an accordion </section></blaze-accordion-pane> </div></blaze-accordion>"`;

exports[`two headers and two panes 2`] = `"<blaze-accordion class=\\"hydrated\\"><!----><div class=\\"c-card c-card--accordion\\"> <blaze-accordion-pane open=\\"\\" header=\\"Click me\\" class=\\"hydrated\\"><!----><button role=\\"heading\\" aria-expanded=\\"true\\" class=\\"c-card__control c-card__control--active\\">Click me</button><section class=\\"c-card__item c-card__item--pane\\"> This is an accordion </section></blaze-accordion-pane> </div></blaze-accordion>"`;
